### ISSUE TITLE
Panic: a query with now()

### ISSUE THREAD

**Issue by lmatz** [risingwavelabs/risingwave#7131]
### Describe the bug

commit: 85aab8cb08c79a1dda89f51534c7b64cfbbf245f

```
create table t(v1 int, t1 timestamp); 
create materialized view mv as select * from t where t.t1 > now() - INTERVAL '1 hour';
```
Is it a known issue due to https://github.com/risingwavelabs/risingwave/issues/6813?
But probably shouldn't panic at assertion anyway?

```
2022-12-29T14:19:31.388810Z  INFO risingwave_meta::hummock::compaction_scheduler: Start compaction scheduler.
  16 2022-12-29T14:19:33.609951Z  INFO risingwave_meta::hummock::compactor_manager: Added compactor session 1
  17 thread 'risingwave-main' panicked at 'assertion failed: `(left == right)`
  18   left: `3`,
  19  right: `2`', src/meta/src/rpc/service/ddl_service.rs:547:9
```

[meta-node-5690.log](https://github.com/risingwavelabs/risingwave/files/10319924/meta-node-5690.log)

[compute-node-5688.log](https://github.com/risingwavelabs/risingwave/files/10319925/compute-node-5688.log)

[frontend-4566.log](https://github.com/risingwavelabs/risingwave/files/10319926/frontend-4566.log)

**Comment by soundOfDestiny** [comment]
> Is it a known issue due to https://github.com/risingwavelabs/risingwave/issues/6813?

I think that's correct

**Comment by soundOfDestiny** [comment]
solved by #7148
